pkgs/development/libraries/qt-5/hooks/qtbase-setup-hook.sh

```diff
@@ -1,3 +1,13 @@
+if [ -z "$_qt_mkDerivation" ]
+then
+    cat >&2 <<EOF
+Did not detect supported mkDerivation. Build aborted!
+
+Please refer to the Nixpkgs manual section 'Support for specific programming languages and frameworks/Qt'.
```

Comment:
Not a blocker issue. Is there something we can do to improve the message in some way?

Linking to the manual page should be possible, even if not yet online (will be when 19.09 is released). All sections should already have stable IDs, it's a matter of placing it after `https://nixos.org/manual/nixpkgs/`.

Maybe add some keywords to help a forgetful person? Something like "tip: look for qt5.callPackage" or something like that.